umm-spec-lib/resources/json-schemas/collection/umm/v1.18.0/samples/ecse-1473.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Delete this file, it should not exist in the CMR, unless it is used for testing, Then it should be moved under the example records directory.